scala3doc-testcases/src/tests/hkts.scala

```diff
@@ -0,0 +1,6 @@
+package tests
+package hkts
+
+class A[F[_]]
+
+// class B[[A, B] =>> (A, B)]
```

Comment:
Why do we need that comment? If this does not work probably we need an issue for it

Reply:
It's leftover, shouldn't be included in that PR at all, will remove both